### DISCUSSION TITLE
result.result must not be null / Appium -js

### DISCUSSION THREAD

**Discussion by byKosta** [appium/appium#19593] [General]
When running app on a real device, everything works without problems.

click Login > I have successfully logged into the app

I'm running the app on the emulator: click Login > result.result must not be null

Updated capabilities

added > const chromeOptions = {

         'appium: chromeOptions' : {

         w3c: false };

but it didn't help.

I didn't find enough information to solve problem and I'll ask for help here. According to information i found, there may be a problem in capabilities. Anyone who has encountered this problem please respond

**Comment by jlipps** [comment]
Please fill out an issue report and attach all requested info there.
